R/sample_predictions.R

```diff
@@ -46,6 +49,7 @@
 sample_prediction <- function(point_nowcast_matrix,
                               reporting_triangle,
                               dispersion,
+                              observation_model = "negative binomial",
```

Comment:
the pass in here should be a function i.e your fitting function and similar.

Reply:
Yes need to update this